### PR TITLE
<press-bind>

### PR DESCRIPTION
**Pull request by ianwremmel**

### CODE REVIEW

File: features/server/views/pages/bind.ejs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<section class="ui basic segment">
+  <header>
+    <h1>PRESS Test Site</h1>
+    <h2>Date Range Picker</h2>
+  </header>
+
+  <main class="ui two column grid">
+    <div class="ui column">
+      <h3>Code Under Test</h3>
+      <press-bind id="notnested" name="email" value="unnested@example.com"></press-bind>
```
Comment (bzang) [inline features/server/views/pages/bind.ejs]: Does the `id` matter here? It seems to be used to select the DOM element below, but the docs seem to suggest that it doesn't really matter and all the happens is that `email` gets added to the vue model.

Comment (bzang) [reply features/server/views/pages/bind.ejs]: also, maybe `dataPath` instead of `name`?

Comment (ianwremmel) [reply features/server/views/pages/bind.ejs]: yea, `id` is just here to support the tests, it's not relevant to functionality

**Comment by ianwremmel** [comment]
> Overall seems reasonable. Main thought I had was... can't something like this be used outside of a form since it's mainly to add server side data to the vue model without making an extra request?

There's nothing that requires this be part of a form. Though they are `input` elements, the test code works outside a form. I checked the standard and `<input>`s can be used anywhere phrasing content is allowed, not just as a child of `<form>`. It doesn't quite sit right with me, but it seems to be ok. 

> If so, are single fields useful enough? Could we instead create a helper that generates an initial json payload on the page and then merge that into the vue model? Maybe this is a different use case.

yea, i was thinking of adding that, but I'm on the fence. I feel like we don't want to make it *too* easy to push a whole lot of data down. that said, it should be pretty straightforward to support JSON

File: features/server/views/pages/static-value.ejs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  <header>
+    <h1>PRESS Test Site</h1>
+    <h2>Static Value</h2>
+  </header>
+
+  <main class="ui two column grid">
+    <div class="ui column">
+      <h3>Code Under Test</h3>
+      <press-static-value id="notnested" press-key-path="email" press-value="unnested@example.com"></press-static-value>
+      <press-static-value id="nested" press-key-path="user[email]" press-value="nested@example.com"></press-static-value>
```
Comment (marceloeloelo) [inline features/server/views/pages/static-value.ejs]: Do we need to test for double nestings? `"user[address][city]"` ?

**Pull request merged**